viewing_party/party.py

```diff
@@ -1,23 +1,188 @@
 # ------------- WAVE 1 --------------------
+# test_wave_01 - line 7 to line 59
+from logging import StringTemplateStyle
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    new_movie = {'title':title,
+    'genre': genre,
+    'rating': rating}

```

Comment:
This is a stylistic choice you will see often. 

```suggestion
    new_movie = {
        'title':title,
        'genre': genre,
        'rating': rating
       }
```